src/chocolatey/infrastructure.app/configuration/PackagesConfigFilePackageSetting.cs

```diff
@@ -50,9 +50,6 @@ public sealed class PackagesConfigFilePackageSetting
         [XmlAttribute(AttributeName = "forceX86")]
         public bool ForceX86 { get; set; }
 
-        [XmlAttribute(AttributeName = "allowMultipleVersions")]

```

Comment:
Is it worth keeping this around, but emit a warning that the attribute is no longer supported?

Reply:
To not break the schema of Chocolatey `packages.config` files. The deprecation warnings only mentioned the actual side by side installs, so I think a user could have an expectation of having existing `packages.config` files still work, while ignoring/warning on that attribute.

Reply:
Assuming that the reading of the `packages.config` happens the same way that the config file and some Intune files happens, then the presence of the attribute in the file will not cause any issues as long as it's a valid xml document. This of course is something that we should test out of course (once I've gotten a build off of this PR I'll give it a try and can report back.)

Reply:
Yeah, as long as the xml is valid, it loads just fine, even if it has attributes and elements that aren't part of the class.

![2023-03-17_19-17-52](https://user-images.githubusercontent.com/30301021/226078046-c8e360ae-55c2-4320-a1b7-c0103e4ad9ba.png)

As with the other comment, I'd leave it up to @gep13 to determine if we should keep it or not.

Reply:
Unless there is a technical reason this should be kept, we need to remove it as it's no longer needed.

Reply:
I would agree that it can be removed completely since then the attribute would be ignored.

Reply:
I am ok with this being removed.